actpool/actpool.go

```diff
@@ -226,6 +238,22 @@ func (ap *actPool) GetCapacity() uint64 {
 	return ap.cfg.MaxNumActsPerPool
 }
 
+// GetGasSize returns the act pool gas size
+func (ap *actPool) GetGasSize() uint64 {
+	ap.mutex.RLock()
+	defer ap.mutex.RUnlock()
+
+	return ap.gasInPool
+}
+
+// GetGasCapacity returns the act pool gas capacity
+func (ap *actPool) GetGasCapacity() uint64 {
+	ap.mutex.RLock()
```

Comment:
This lock is not necessary as MaxGasLimitPerPool should behave as an immutable val